### PR TITLE
[12.0] [MIG] Account Payment Return

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Migration of `account_payment_return`.

Migrations of `account_payment_return_import`: #215 
Migrations of `account_payment_return_import_sepa_pain`: #216

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please only one module per PR.

**Review by pedrobaeza — APPROVED** [review]

File: account_payment_return/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'license': 'AGPL-3',
+    "depends": [
+        'mail',
+        'account',
+    ],
+    'author': '7 i TRIA, '
+              'Tecnativa, '
+              'initOS GmbH & Co., '
+              'Odoo Community Association (OCA)',
+    'website': 'https://www.tecnativa.com',
```
Comment (Nikul-Chaudhary) [inline account_payment_return/__manifest__.py]: @ThomasBinsfeld URL should be `https://github.com/OCA/account-payment`

File: account_payment_return/data/ir_sequence_data.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     Copyright 2015 Sergio Teruel <sergio.teruel@tecnativa.com>
+     Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+<odoo noupdate="1">
+    <data>
```
Comment (Nikul-Chaudhary) [inline account_payment_return/data/ir_sequence_data.xml]: no need `data`

File: account_payment_return/models/account_invoice.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        returned_invoices = self.env['account.partial.reconcile'].search(
+            [('origin_returned_move_ids.invoice_id', 'in', self.ids)]).mapped(
+            'origin_returned_move_ids.invoice_id')
+        returned_invoices.filtered(
+            lambda x: not x.returned_payment).write(
+            {'returned_payment': True})
+        (self - returned_invoices).filtered('returned_payment').write(
+            {'returned_payment': False})
+
+    @api.one
```
Comment (Nikul-Chaudhary) [inline account_payment_return/models/account_invoice.py:30]: `@api.one` ??

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improved Code

File: account_payment_return/models/account_invoice.py (hunk 3 above)
Comment (ThomasBinsfeld) [reply account_payment_return/models/account_invoice.py:30]: @Nikul-Chaudhary The super method is `@api.one`. I prefer keep it this way.

File: account_payment_return/data/ir_sequence_data.xml (hunk 2 above)
Comment (ThomasBinsfeld) [reply account_payment_return/data/ir_sequence_data.xml]: Done

File: account_payment_return/__manifest__.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply account_payment_return/__manifest__.py]: Done

**Comment by ThomasBinsfeld** [comment]
@Nikul-Chaudhary Code updated. Thanks for your review.

**Review by carlosdauden — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**